src/utils/luminance.js

```diff
@@ -0,0 +1,12 @@
+import parseColor from 'parse-color';
+
+export const getLuminance = (color) => {
+	const [r, g, b] = parseColor(color).rgb;
```

Comment:
There was a bug here as well, this used to say
```js
const [r, g, b] = parseColor(color).rgb;
```
but that returned `{undefined, undefined, undefined }`